src/lib/core/overlay/position/viewport-ruler.ts

```diff
@@ -55,4 +66,13 @@ export class ViewportRuler {
 
     return {top, left};
   }
+
+  /**
+   * @internal
+   * Updates the cached client rectangle of the document element.
+   */
+  updateDocumentRectangle() {
```

Comment:
I think our convention is to name internal methods with _, if that's not possible we add a @docs-private comment. IIRC marking stuff @internal caused issues in google3